dts/arc/yaml/intel,qmsi_ss-uart.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
either the filename should be intel,qmsi-uart.yaml or the Title, id, compatible constraint all need fixing

Reply:
k